manifests/kustomize/base/kustomization.yaml

```diff
@@ -28,3 +29,7 @@ images:
   newTag: 0.2.4
 - name: gcr.io/ml-pipeline/metadata-writer
   newTag: 0.2.4
+- name: gcr.io/ml-pipeline/execution-cache
+  newTag: 0.2.4
+- name: gcr.io/ml-pipeline/execution-cache-deployer
+  newTag: 0.2.4
```

Comment:
1. by the way, did you test it in standalone installation? aka install via kubectl with these yamls?
2. gcp_marketplace should be updated to include this new feature. However we may solve 1 first as MKP Beta is on releasing.

Reply:
1. Yes I tested it in standalone installation. Also when running e2e tests and sample tests the cache service works good.
2. Manifests for MKP is also ready in my local branch. Considering MKP Beta is on releasing, can we make that a seperate PR?

Reply:
2. sure, I also prefer separate PR. Thanks.

Reply:
LGTM this PR but didn't leave label (to let others continue the reviews)